tests/robot-tests/tests/libs/common.robot

```diff
@@ -559,13 +584,15 @@ user opens details dropdown
     [Arguments]    ${text}    ${parent}=css:body
     user waits until parent contains element    ${parent}
     ...    xpath:.//details/summary[contains(., "${text}") and @aria-expanded]    60
+    ${details}=    get child element    ${parent}    xpath:.//details[summary[contains(., "${text}")]]
```

Comment:
I've not seen this before. I assume this fetches the details tag that contains a summary that contains the text?

Reply:
Yup that's right!